ui/src/Components/ProfilePage.jsx

```diff
@@ -112,39 +133,83 @@ export default class ProfilePage extends React.Component {
           {!this.state.profile && this.state.loading == false ? (
             <div> Profile Does Not Exist </div>
           ) : (
-            <div className="row-buffer side-buffer left-align">
-              <h4>{this.state.profile.result.first_name} {' '}
-              {this.state.profile.result.last_name}</h4>
-              {this.state.profile.result.city ? <H4>Servicing the {this.state.profile.result.city} area</H4> : <div/>}
-              <Table className="row-buffer" responsive>
+            <div className="row-buffer side-buffer">
+               {this.state.profile.result.app_status == 'Began Application' ? (
+                  <Alert className="width-fit" color="primary">
+                    Your application is in draft mode.
+                    Submit your application to employers through Edit Profile > Submit
+                  </Alert>                    
+                  ) : <div/>}
+                <Alert className="push-right" color="success">
+                  APPLICATION STATUS: {this.state.profile.result.app_status}
+                </Alert>
+                <h4 className="push-left">{this.state.profile.result.first_name} {' '}
+                {this.state.profile.result.middle_name} {' '}
+                {this.state.profile.result.last_name}</h4>
+              <div>
+              </div>
+              <Table responsive>
                 <tbody>
                 <tr>
-                    <td>Email</td>
+                    <th>Email</th>
                     <td>{this.state.profile.result.email}</td>
                   </tr>
                   <tr>
-                    <td>Phone Number</td>
-                    <td>{this.state.profile.result.contact_num}</td>
+                    <th>Phone Number</th>
+                    {this.state.profile.result.contact_ext ? (
+                    <td>{this.state.profile.result.contact_num.substring(0,3)}-{this.state.profile.result.contact_num.substring(3,6)}-{this.state.profile.result.contact_num.substring(6,10)}  x  {this.state.profile.result.contact_ext}</td>
```

Comment:
Yeah, the fact that we do the same thing more than once really makes me think that we should create a method to format the phone numbers properly.

Reply:
Just to call out, this is expecting all numbers to be US/Can for now... can we add that as a comment that this is our current assumption. No need to change the logic bc I doubt this will be WorldWide anytime soon

Reply:
Yes exactly @nicolasabousawan ! Especially since our create profile phone number field only accepts 10 digit phone numbers, which is consistent with US/Can but not world wide

Reply:
@kvptkr Just pushed with formatter functions added :)